RvmSharp/Primitives/RvmFacetGroup.cs

```diff
@@ -12,4 +13,19 @@ RvmFacetGroup.RvmPolygon[] Polygons
     public record RvmContour((Vector3 Vertex, Vector3 Normal)[] Vertices);
 
     public record RvmPolygon(RvmContour[] Contours);
+
+    public RvmBoundingBox CalculateBoundingBoxFromVertexPositions()
```

Comment:
Add an xml comment to this method. Consider adding a `<remark>` as to when it could be useful.